packages/kit/src/node/index.js

```diff
@@ -2,51 +2,41 @@ import * as set_cookie_parser from 'set-cookie-parser';
 
 /** @param {import('http').IncomingMessage} req */
 function get_raw_body(req) {
-	return new Promise((fulfil, reject) => {
-		const h = req.headers;
+	const h = req.headers;
 
-		if (!h['content-type']) {
-			return fulfil(null);
-		}
+	if (!h['content-type']) {
+		return null;
+	}
 
-		req.on('error', reject);
+	const length = Number(h['content-length']);
 
-		const length = Number(h['content-length']);
+	// https://github.com/jshttp/type-is/blob/c1f4388c71c8a01f79934e68f630ca4a15fffcd6/index.js#L81-L95
```

Comment:
we could expand the comment to make it clearer what this does without needing to follow the link
```suggestion
	// check if no request body
	// https://github.com/jshttp/type-is/blob/c1f4388c71c8a01f79934e68f630ca4a15fffcd6/index.js#L81-L95
```